README.md

```diff
@@ -72,26 +188,37 @@ To enable it locally outside of the Docker container:
     - `cd nsjail && make`
 - Enable `unprivileged_userns_clone`
     - Temporary: `sudo sysctl -w kernel.unprivileged_userns_clone=1`
-    - Permanent: `echo 'kernel.unprivileged_userns_clone=1' | sudo tee -a /etc/sysctl.d/00-local-userns.conf && sudo service procps restart
-`
-- Edit `.env`:
+    - Permanent: `echo 'kernel.unprivileged_userns_clone=1' | sudo tee -a /etc/sysctl.d/00-local-userns.conf && sudo service procps restart`
+
+- Edit `local.env`:
     - Set `USE_SANDBOX_JAIL=on`
     - Set `SANDBOX_NSJAIL_BIN_PATH` to the absolute path of the `nsjail` binary built above
 
-### Frontend
+## Contributing
 
-You'll need [Yarn](https://yarnpkg.com/getting-started/install) (which requires Node.js).
+Contributions are very much welcome! We have a Discord channel in the [Zelda Decompilation server](https://discord.gg/k9gY7uCysW).
```

Comment:
I think it'd be better to use discord.zelda64.dev here in case we change the link at some point